docs/contributing/working.md

```diff
@@ -1,26 +1,24 @@
 ---
 id: working
-title: Contributing
+title: Working
 sidebar_label: Working
 ---
 
-# Working
-
 ## Managing Issues
 
 Well crafted issues improve the speed that work can be triaged, understood, and completed. To help guide this process [issue templates](https://github.com/Microsoft/fast-dna/tree/master/.github/ISSUE_TEMPLATE) exist to allow users to choose an appropriate [bug report](https://github.com/Microsoft/fast-dna/blob/master/.github/ISSUE_TEMPLATE/Bug_report.md), [feature request](https://github.com/Microsoft/fast-dna/blob/master/.github/ISSUE_TEMPLATE/Feature_request.md), or [custom template](https://github.com/Microsoft/fast-dna/blob/master/.github/ISSUE_TEMPLATE/Custom.md).
```

Comment:
Since we're in here, these links are wrong, looks like we have two markdowns and all of these will 404, see https://github.com/Microsoft/fast-dna/tree/master/.github/ISSUE_TEMPLATE